examples/v1/pipelineruns/alpha/pipelinerun-with-matrix-context-variables.yaml

```diff
@@ -0,0 +1,95 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: echomatrixlength
+spec:
+  params:
+    - name: matrixlength
+      type: string
+  steps:
+    - name: echo
+      image: alpine
+      script: echo $(params.matrixlength)
```

Comment:
same as below, we can confirm the expected length such that it works as a real test along with an example